pkg/services/sqlstore/org_test.go

```diff
@@ -35,6 +35,44 @@ func TestAccountDataAccess(t *testing.T) {
 			So(len(query.Result), ShouldEqual, 3)
 		})
 
+		Convey("Given we have organizations, we can limit and paginate search", func() {
+			var err error
```

Comment:
I would just remove this variable and define the error variable within the individual test case functions, since we're not going to use the variable outside of them. I see it's copied from the old test, but I think it could be better.